addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -132,66 +149,82 @@
 "chart_at_template_3542","Verrechnung Dienstgeberzuschlag","3542","True","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
 "chart_at_template_3550","Verrechnung Kommunalsteuer","3550","False","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
 "chart_at_template_3551","Verrechnung Wiener Dienstgeberabgabe","3551","False","liability_current","l10n_at.account_tag_l10n_at_PCVIII1"
-"chart_at_template_3600","Verrechungskonto Sozialversicherung","3600","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
-"chart_at_template_3610","Verrechnungskonto Magistrat/Gemeinde (KoSt, U-Bahn, etc.)","3610","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
+"chart_at_template_3600","Verbindlichkeiten im Rahmen der sozialen Sicherheit","3600","True","liability_payable","l10n_at.account_tag_l10n_at_PCVIII2"
 "chart_at_template_3740","WERE Verrechnungskonto","3740","True","liability_payable","l10n_at.account_tag_l10n_at_PD"
 "chart_at_template_4000","Brutto-Umsatzerlöse im Inland (20%)","4000","False","income","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
 "chart_at_template_4001","Brutto-Umsatzerlöse im Inland (10%)","4001","False","income","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
 "chart_at_template_4100","Brutto-Umsatzerlöse im EU Raum (RC 20%)","4100","False","income","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
 "chart_at_template_4110","Brutto-Umsatzerlöse im EU Raum (RC 10%)","4110","False","income","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
 "chart_at_template_4200","Brutto-Umsatzerlöse in Drittstaaten (0%)","4200","False","income","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
 "chart_at_template_4860","Kursgewinne aus Fremdwährungstransaktionen","4860","False","income_other","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT1"
-"chart_at_template_5000","Wareneinkauf","5000","False","expense","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT5I"
-"chart_at_template_5050","Wareneinkauf EU","5050","False","expense","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT5I"
+"chart_at_template_5000","Wareneinsatz","5000","False","expense","account.account_tag_operating,l10n_at.account_tag_l10n_at_EBIT5I"
```

Comment:
I think we should replace "expense" with "cost of goods sold" in the whole class 5